base/iterators.jl

```diff
@@ -1343,4 +1344,144 @@ only(x::NamedTuple) = throw(
     ArgumentError("NamedTuple contains $(length(x)) elements, must contain exactly 1 element")
 )
 
+
+"""
```

Comment:
I think people would expect to find this in range.jl instead.

Reply:
OK, I have moved it back now.